src/pipeline/bundle-js.js

```diff
@@ -39,6 +39,10 @@ module.exports = function (opts, paths, output) {
       fullPaths: true,
       cacheFile: path.join(paths.TMP_DIR, 'browserify-cache.json'),
       transform: getTransform(opts),
+      paths: [
+        path.join(paths.INPUT_DIR, 'node_modules'),
```

Comment:
@mathisonian will this cause an issue if the input dir doesn't have a node_modules folder?